modules/oaiharvest/lib/oai_harvest_daemon.py

```diff
@@ -572,36 +572,47 @@ def remove_duplicates(harvested_file_list):
     """
     Go through a list of harvested files and remove any duplicate records.
     """
+    from copy import deepcopy
+    import lxml.etree as etree
```

Comment:
Understood. Thats why I asked in chat if lxml is a good way to go.

To solve it without parsing xml would require to go with regexping again. I'm not sure it's easy to come up with a general regexp here. Is an other way of xml parsing preferable? Or do you think one should just raise an error notifying admins to get lxml as a solution?